examples/gallery/grid/grdview_surface.py

```diff
@@ -4,11 +4,11 @@
 
 The :meth:`pygmt.Figure.grdview()` method can plot 3-D surfaces with ``surftype="s"``. Here,
 we supply the data as an :class:`xarray.DataArray` with the coordinate vectors ``x`` and
-``y`` defined. Note that the ``perspective`` argument here controls the azimuth and
+``y`` defined. Note that the ``perspective`` parameter here controls the azimuth and
 elevation angle of the view. We provide a list of two arguments to ``frame`` — the
 second argument, prepended with ``"z"``, specifies the :math:`z`-axis frame attributes.
```

Comment:
Not strictly necessary or related to parameter vs argument, but a small improvement here would be to specify that the first argument specifies the x- and y- axis frame attributes for completeness.

Reply:
@meghanrjones Not sure what you're recommending; could you please explain further?